docs/mvc/baseController.markdown

```diff
@@ -0,0 +1,68 @@
+---
+layout: default
+title: Base Controllers
+name: Overview
+parent: Model View Controller
+nav_order: 1
+---
+
+# Use base controllers
+
+Keep commonly used methods such as
+
+- getResourceBundle
+- getRouter
+- getModel
+- setModel
+- onNavBack
+- _or whatever method or property is being actively re-used inside your project_
```

Comment:
wording suggestion: "method or property is being re-used in other controllers in the application".
but YMMV :)

Reply:
done